palisade-service/src/main/resources/application-h2.yaml

```diff
@@ -25,4 +25,4 @@ spring:
   jpa:
     hibernate:
       ddl-auto: create-drop
-    show-sql: false
+    show-sql: true
```

Comment:
Recommended not show the sql, it isn't terribly meaningful and is just messy

Reply:
Did not manually change this.  Think that it came into the code via a merge, but more important the develop branch has it set to false.  I'll change it back to false.